src/cluster.py

```diff
@@ -124,6 +124,9 @@ def stale_quorum(self) -> bool:
         if not self.all_units_related:
             return False
 
+        if self.relation.data[self.charm.app].get("changed-quorum", None):
+            return True
+
```

Comment:
Although it was my omission in the VM charm, this should probably use `self.changed_quorum`